_docs/configure/index.md

```diff
@@ -1,5 +1,48 @@
 ---
-title: Configure 
+title: Configuring Forseti
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This page describes how to configure Forseti server once it's set up. Forseti configurations
+are global and module-specific settings such as the following:
+
+-   `global`: configurations that are used by multiple modules such as SendGrid API key,
+    GSuite admin account and email recipients.
+-   `scanner`({% link _docs/configure/scanner/index.md %}): configurations that are used only by Forseti Scanner, such as
+    specifying which scanners to enable.
+-   `notifier`({% link _docs/configure/notifier/index.md %}): configurations that are used only by Forseti Notifier, such as
+    specifying which notifications to enable.
+
+Configurations are centrally maintained in
+forseti-security/configs/server/forseti_server_conf.yaml file that's organized into
+module-specific sections.
+
+## Configuring settings
+
+To set up your configuration, you'll edit
+[forseti_server_conf.yaml.sample](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/configs/server/forseti_server_conf.yaml.sample)
```

Comment:
`htmlproofer` can't find this file https://github.com/GoogleCloudPlatform/forseti-security/blob/master/configs/server/forseti_server_conf.yaml.sample

Reply:
Link updated.